### PR TITLE
clarify what's required for transport without native pub-sub

### PR DESCRIPTION
**Pull request by weralabaj**
Resolves https://github.com/Particular/docs.particular.net/issues/860

### CODE REVIEW

File: nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed.md [hunk 1]
```diff
@@ -16,9 +16,14 @@ Messages matching the following criteria will be auto subscribed at startup.
...
-### Routing configuration needed
+### Transports without native support for publish-subscribe
 
-If the selected transport doesn't natively supports publish-subscribe it is required to specify the address of the publisher for each event. This is done by [specifying a message owner in the message endpoint mappings](/nservicebus/messaging/message-owner.md). For example:
+In case of [transports without native support for publish-subscribe](/nservicebus/messaging/publish-subscribe/#mechanics-persistence-based) it is necessary to satisfy additional conditions:
+
+ 1. The project needs to use persistence, as explained in [Persistence Based Publish Subscribe](/nservicebus/messaging/publish-subscribe/#mechanics-persistence-based) section.
+ 1. An address of the publisher needs to be specified for each event, as described below.
+
+The publisher's address is provided by [specifying a message owner in the message endpoint mappings](/nservicebus/messaging/message-owner.md). For example:
```
Comment (SimonCropp) [inline nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed.md]: this should not exist under the "Transports without native support for publish-subscribe" heading

File: nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed.md [hunk 2]
```diff
@@ -16,9 +16,14 @@ Messages matching the following criteria will be auto subscribed at startup.
  1. At least one [message handler and/or saga](/nservicebus/handlers/) exists for the given message
 
 
-### Routing configuration needed
+### Transports without native support for publish-subscribe
```
Comment (SimonCropp) [inline nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed.md]: does this really belong in a page title "Controlling what is subscribed"?

File: nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed.md (hunk 1 above)
Comment (weralabaj) [inline nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed.md]: which part? the second point was there already, the first I took from http://docs.particular.net/nservicebus/messaging/publish-subscribe/, so if something is inaccurate we need to fix existing docs

File: nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed.md (hunk 2 above)
Comment (weralabaj) [inline nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed.md]: move to http://docs.particular.net/nservicebus/messaging/publish-subscribe/?

Comment (SimonCropp) [inline nservicebus/messaging/publish-subscribe/controlling-what-is-subscribed.md]: i think so

**Comment by weralabaj** [comment]
@SimonCropp better now?

**Pull request merged**